frontend/integration-tests/protractor.conf.ts

```diff
@@ -31,7 +31,7 @@ export const config: Config = {
   skipSourceMapSupport: true,
   jasmineNodeOpts: {
     print: () => null,
-    defaultTimeoutInterval: 40000,
+    defaultTimeoutInterval: 60000,
```

Comment:
what is this timeout for? Can you set it for the cases where this is used? Probably not a good idea to set it globally.

Reply:
I think it's jasmine convention to have a global timeout, it apply to all async operations.
When I run testing locally, some test fails because the time it takes exceeding this default timeout, so I increase it.

Reply:
@rhrazdil can you please check if this is a good idea?

Reply:
@suomyi @gouyang Please don't do this. Any TCs that need specifically more time should have it set locally, global timeout can never accommodate all TCs anyway.

To clear up thw random fails; I know about the issue with vm actions cases that fail sometimes.
The problem isn't timeout though, it's some random issue with dropdowns, I haven't figured it out yet.

Reply:
There are some random fails like `Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.` without a clear fail step, but just the default timeout, so I'm trying to increase it. And it seems related to `vm.action` step which has the dropdowns. 

I changed it to the previous one.

Reply:
> The problem isn't timeout though, it's some random issue with dropdowns, I haven't figured it out yet.

Do these occur also in vm detail? I noticed some problems with vm rows only so far.

Reply:
@suomiy It mostly happens when start/restart/delete a VM either from VM list page or VM detail page.

It even occurs in `Logs in` sometimes, increase the timeout could improve this situation from what I've seen, especially when I run testing code on my laptop with a remote cluster.

```
Failed Specs:

1) Authentication : Logs in.
   Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
       at ontimeout timers.js:436:11
       at tryOnTimeout timers.js:300:5
       at listOnTimeout timers.js:263:5
       at Timer.processTimers timers.js:223:10
```